### DISCUSSION TITLE
Doesn't date format localization not change based on country?

### DISCUSSION THREAD

**Discussion by thibaultmol** [community/community#108161] [Other Feature Feedback, Questions, & Ideas] [answered]
In the "New fine-grained personal access token" section of your github accounts settings, the date is in the 'US' date format. But I live in Belgium and have that set on my github account.
I don't think i've ever seen MM/DD/YYYY anywhere on Github. it's normally DD/MM/YYYY
![image](https://github.com/community/community/assets/1010226/3de779d2-837a-4db3-8b4a-c00a9ea7f883)

**Comment by jmarintur** [comment, marked as answer]
Hi @thibaultmol, it's worth a try setting your browser and computer to Belgian settings to see if you get dates in the DD/MM/YYYY format on GitHub. I don't think GitHub lets you change date formats directly.

**Reply by thibaultmol** [reply]
OH YEAH, that's it.... I'm on new install and the locale was incorrect on there. Sorry, thank you!
